src/data/markdown/docs/01 guides/05 Testing Guides/02 Load testing websites.md

```diff
@@ -3,115 +3,99 @@ title: "Load testing websites"
 
-A test that steps through different levels of load and will highlight where performance problems start to happen.
-You should expect to iterate this test multiple times as you enter a test-> analyze-> make changes-> repeat pattern.
+> Start small and simple, make sure you get something out of the testing first, then expand the test suite and add more complexity until you feel that you’ve reached the point where more effort spent on realism will not give enough return on your invested time.
+> 
+> 
+> [Simple testing is better than no testing](https://k6.io/our-beliefs#simple-testing-is-better-than-no-testing)
 
+The first thing is to decide what to load test. On one side, you could evaluate which are the performance aspects that are most valuable to your business and has the most significant risks. On the other side, you could identify what your users do regularly. Use your analytics tools to find the most frequent user journeys.
 
-### Load tests
+With this information, it’s time to analyze the frequency of usage, business value, performance risks, and any other critical performance aspect of your organization to help you deciding what to load test first.
 
-After you experience a stable stress test result, this test will become part of your load tests and you should regularly run them to confirm the system can meet your performance goals.
+### Calculate the number of concurrent users
 
-You could also define your load tests based on the observation of your normal and peak traffic levels. You should use data on how many users you have normally and any business goals you may have for concurrency.
+You could define your load tests based on the observation of your normal and peak traffic levels, and use this formula to determine the number of VUs to specify in your load tests:
 
-Use the formula:
+> VUs = (hourly sessions \* average session duration in seconds)/3600
+>
+> - hourly sessions = Number of sessions per hour
+> - average session duration = Average time between the first and last request of each user
 
-> VUs = (Peak Hourly Sessions * Average Session Duration in Seconds) / 3600
+Refer to [Determining Concurrent Users in Your Load Tests](https://k6.io/blog/monthly-visits-concurrent-users) to learn how to calculate the formula using Google Analytics.
 
-You should run your test long enough for all VUs to complete 2-5 complete iterations of your test script. If your user journey takes 5 minutes to complete, 10+ minutes would be a reasonable test length.
 
-### Other tests as required
+### Load test your website in pre-production.
 
-You may have other traffic patterns you want to test, for example, from different load zones or different ramping profiles. Continue testing here as needed.
+We recommend to <a href="https://k6.io/our-beliefs#load-test-in-a-pre-production-environment">Load test in a pre-production environment</a> that mimics the production environment as closely as possible.
 
-### Continuous and Regression Testing
```

Comment:
I think we should still include this paragraph

Reply:
The whole section was:

> Which type of performance tests should I run against my website?

I have removed this because it is better covered with the `Test Types` guide.